_posts/2025-03-26-kubeflow-1.10-release.md

```diff
@@ -0,0 +1,268 @@
+---
+title: "Kubeflow 1.10 Release Announcement"
+layout: post
+toc: false
+comments: true
+image: images/logo.png
+hide: false
+categories: [release]
+permalink: /kubeflow-1.10-release/
+author: "Kubeflow 1.10 Release Team, Dimitris Poulopoulos"
+---
+
+Kubeflow 1.10.0 delivers essential updates that enhance the flexibility, efficiency, and scalability of machine learning
+workflows. The new features span across several components, improving both user experience and system performance.
+
+## Benefits to the ecosystem
+
+* Migration to GitHub Container Registry (ghcr) as default container registry for (most) images
```

Comment:
Should we highlight new initiatives here like ML Experience WG, Kubeflow SDK, Kubeflow Outreach Committee, Kubeflow Helm Charts, Kubeflow Feast and new project adoption process.
@kubeflow/kubeflow-steering-committee @jbottum Am I missing something? 
I think, KF 1.10 release is a good way to expose this to the wider community.

Reply:
I like those projects and they attract new members.  This post is about 1.10 (and is long).   We should probably recap 1.10 1st and then (if not too long) add after 1.10 summary about the new / enhanced projects in this list.

Reply:
+1

Reply:
Any discussion of things which are pending approval or under discussion needs to be carefully worded to not imply that the decision is final, or that we are 100% committed to it.

For example, Feast is not formally part of the project yet, and Helm Charts have massive debate around what form they will take (e.g. standalone charts vs platform).

Reply:
I also believe that this post is already too long to add anything to it.

Reply:
I agree. It makes the blog longer without adding much.

Reply:
yes, I agree.